### PR TITLE
Add model options

### PR DESCRIPTION
**Pull request by CA-rvinson**
@genkikondo I'm working on trying to standardize the models so the control code can use a torch mask RCNN or a YOLO model interchangeably. I've run into some package structure issues that I think you might have some insight into. The main 2 issues are 
- What would be the best way for the camera_node or control_node to be able to import ml_models. In theory we could build a package via at setup.py script and manually pip install it, but that seems significantly more round about then it feels like it needs to be. 
- The pytorch mask-rcnn model weights are larger then github allows. I thought about putting them in the ml_model data_store folder and then using trying to use get_package_share_directory to pull the location, but these seems like it might have other issues. It would also require pulling from dvc in production systems, which might not be desirable long term.
Let me know if you have any thoughts or suggestions about either point

### CODE REVIEW

**Comment by genkikondo** [comment]
The challenge is how ROS expects packages to be installed. We have to either 1) pip install the local python package into the venv, 2) somehow include the python package into the ROS package install (though this would result in duplicates if multiple ROS packages depended on the same python package), 3) use the ROS dependency system, which would mean we'd need to turn the python package into a ROS package. I'd try #1 before #3, but either is fine with me. What python code do we need from ml_models?

re: large model weights size, we'd have to fetch it from cloud storage no matter what (whether thru DVC or some setup script); I personally don't think it's bad to pull from DVC (we can just pull specific subdirs)

File: yolov8n.pt
Comment (genkikondo) [inline yolov8n.pt:1]: do we need to check this in?

Comment (CA-rvinson) [reply yolov8n.pt:1]: nope, it gets pulled to train a yolo model, but I should just add it to the .gitignore file.

**Comment by CA-rvinson** [comment]
Yeah, I think #1 is the best option. It just means we need to add a setup file and then build and locally install it for now
